lasagne/layers/special.py

```diff
@@ -354,3 +356,63 @@ def _meshgrid(height, width):
     ones = T.ones_like(x_t_flat)
     grid = T.concatenate([x_t_flat, y_t_flat, ones], axis=0)
     return grid
+
+
+class ParametricRectifierLayer(Layer):
+    """
+    lasagne.layers.ParametricRectifierLayer(incoming,
+    alpha=init.Uniform((0.001, 0.1)), **kwargs)
+
+    A layer that applies parametric rectify nonlinearity to its input
+    following K He, X Zhang et al. (http://arxiv.org/abs/1502.01852):
```

Comment:
The citation should follow the usual form (put `[1]_` here, then put the citation in a `References` section).